walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/IssuanceRequestContent.kt

```diff
@@ -54,11 +55,11 @@ internal class VerifiedIdRequestContent(
             for (requirementInGroup in requirementsInGroup) {
                 if (requirementInGroup is IdTokenRequirement) {
```

Comment:
Is it possible to do something like 

if (requirementInGroup is IdTokenRequirement) & (requirementInGroup.configuration == IDTOKENHINT_CONFIGURATION)

instead of nested if statements?